helper/src/main.ts

```diff
@@ -37,9 +37,11 @@ async function run(): Promise<void> {
     core.info('_____________ #> curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"')
     core.info('_____________ #> sudo installer -pkg AWSCLIV2.pkg -target /')
     core.info('_____________ #> aws --version')
-    core.info('(once) Step 2: Configure the AWS CLI with your credentials')
+    core.info('(once) Step 2: Configure the AWS CLI with your credentials and the region of your instance')
+    core.info('those information can be found in your aws account, the region of your instance is not necessarily')
+    core.info('where you currently are (for testing we often use the region Ireland')
     core.info('_____________ #> aws configure')
-    core.info('(once) Step 3: Install Session Manager plugin on your machine')
+    core.info('(once) Step 3: Install Session Manager plugin on your machine, python 2.7 is required for this part')
```

Comment:
I believe it's having the "python" binary which is necessary, not a particular version of python, isn't it?

Reply:
with python 3, I got some error while execution the next steps, I have installed python 2 and I worked fine